Makefile

```diff
@@ -0,0 +1,19 @@
+IMAGE ?= prashcr/vim8s
```

Comment:
yeah, I would host the dockerfile in the repo and have it auto-build on the docker hub